### PR TITLE
fix for Unzip package

### PR DESCRIPTION
**Pull request by NathanielRose**
Fix - Script produces an error message: unzip not found.   This command will fail in base WSL an Unzip package is not installed by default.  

closes https://github.com/microsoft/bedrock/issues/1240

### CODE REVIEW

File: tools/prereqs/setup_terraform.sh [hunk 1]
```diff
@@ -40,6 +40,7 @@ fi
 TERRAFORM_VERSION=`curl -L -s https://github.com/hashicorp/terraform/releases/latest | grep archive | grep zip | awk -F"/v" '{print $2}' | awk -F".zip" '{print $1}'`
 curl -LO -s https://releases.hashicorp.com/terraform/$TERRAFORM_VERSION/terraform_"$TERRAFORM_VERSION"_$arch.zip
 
+sudo apt-get install unzip
```
Comment (bnookala) [inline tools/prereqs/setup_terraform.sh]: this will fail on macos as the package manager is homebrew, not apt

Comment (NathanielRose) [reply tools/prereqs/setup_terraform.sh]: Will update

File: tools/prereqs/setup_terraform.sh [hunk 2]
```diff
@@ -30,8 +30,10 @@ cd $tmp_dir
 ostype=`os_type`
 if [ "$ostype" == "linux" ]; then
     arch="linux_amd64"
+    sudo apt-get install unzip
 elif [ "$ostype" == "macos" ]; then
     arch="darwin_amd64"
+    brew install unzip
```
Comment (bnookala) [inline tools/prereqs/setup_terraform.sh]: this looks good - maybe we can create a follow up issue to track that homebrew is a prereq to invoke `brew` - doesn't have to be in this PR.

https://brew.sh/

Comment (NathanielRose) [reply tools/prereqs/setup_terraform.sh]: Recorded here: https://github.com/microsoft/bedrock/issues/1248

**Pull request merged**